_data-prepper/pipelines/configuration/processors/otel-trace-group.md

```diff
@@ -0,0 +1,79 @@
+---
+layout: default
+title: OTel trace group
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# OTel trace group processor
```

Comment:
```suggestion
# `otel_trace_group` processor
```

Reply:
We can't do this in the heading or it will make it display incorrectly, but for all other instances, I'll make this change. Thanks!